src/home_robot_hw/home_robot_hw/env/stretch_pick_and_place_env.py

```diff
@@ -59,11 +61,17 @@ def __init__(
         self.test_grasping = test_grasping
         self.dry_run = dry_run
         self.debug = debug
+        self.visualize_grasping = visualize_grasping
         self.task_info = {}
         self.prev_obs = None
 
-        with open(cat_map_file) as f:
-            self.category_map = json.load(f)
+        if goal_options is not None:
```

Comment:
The goal options don't get passed to OVMM agent, which is where the segmentation happens, so we shouldn't allow `goal_options` to be passed here. The actual goal object name and receptacle names can be passed as arguments already, and if we want to modify the list of receptacles (which is all we need the category map for), then we should directly modify the category map file.

Reply:
yeah i will remove goal_options

Reply:
removed